testnets/pocket/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "pockettestnet",
+  "assets": [
+    {
+      "description": "The native token of Pocket Network",
+      "denom_units": [
+        {
+          "denom": "upokt",
+          "exponent": 0
+        },
+        {
+          "denom": "pokt",
+          "exponent": 6
+        }
+      ],
+      "base": "upokt",
+      "name": "POKT",
+      "display": "pokt",
+      "symbol": "POKT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.svg"
+      },
+      "keywords": ["rpc", "api"],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/pocket/images/pokt-icon.png"
+        }
```

Comment:
We should add the svg as a new object under the `images` array, or else the SVG in logo_URIs will be overwritten with the png by automation.